e2e/smoke_test/smokeTest.spec.js

```diff
@@ -116,14 +116,13 @@ describe('smoke test', () => {
             it('set insulin sensitivites', async () => {
                 await settings.InsulinSensitivities('500');
             });
-            it.skip('set correction range', async () => {
-                //TODO: unable to set this as we can't easily find sepcific values
-                await settings.CorrectionRange({ range: { min: '179', max: '180' } });
+            it('set correction range', async () => {
+                await settings.CorrectionRanges([{ time: '12:00 AM',min: '179', max: '180' }]);
```

Comment:
[nit] missing space before `min`. Same here and the matching code in other files.